README.md

```diff
@@ -1,25 +1,30 @@
-# Pipeline Global Library
+# Pipeline Shared Libraries
 
 When you have multiple Pipeline jobs, you often want to share some parts of the Pipeline
 scripts between them to keep Pipeline scripts [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself).
 A very common use case is that you have many projects that are built in the similar way.
 
-This plugin adds that functionality by creating a "shared library script" Git repository inside Jenkins.
-Every Pipeline script in your Jenkins see these shared library scripts in their classpath.
+This plugin adds that functionality by allowing you to create “shared library script” SCM repositories.
+It can be used in two modes:
+a legacy mode in which there is a single Git repository hosted by Jenkins itself, to which you may push changes;
```

Comment:
So internal would require placing some files somewhere in `JENKINS_HOME` because core doesn't provide some extension for having folders for plugin needs?